File: app/src/main/java/libs/com/rogerio/utils/GamesFX.java
package libs.com.rogerio.utils;

import android.graphics.Paint;


public class GamesFX {
	/**
	 * Sorteia uma nova cor para uma estrela
	 * 
	 * @return
	 */
	public static Paint newColor() {
		int r = (int) (Math.random() * 255);
		int g = (int) (Math.random() * 255);
		int b = (int) (Math.random() * 255);
		Paint p = new Paint();
		p.setARGB(255, r, g, b);
		return p;
	}
	
}

File: app/src/main/java/libs/com/rogerio/utils/GamesFX.kt
package libs.com.rogerio.utils

import android.graphics.Paint


object GamesFX {
    /**
     * Sorteia uma nova cor para uma estrela
     *
     * @return
     */
    fun newColor(): Paint {
        val r = (Math.random() * 255).toInt()
        val g = (Math.random() * 255).toInt()
        val b = (Math.random() * 255).toInt()
        val p = Paint()
        p.setARGB(255, r, g, b)
        return p
    }

}
